test/jira/README.md

```diff
@@ -0,0 +1,65 @@
+# Jira Kamelet test
+
+This test verifies the Jira Kamelet source defined in [jira-source.kamelet.yaml](jira-source.kamelet.yaml)
```

Comment:
wrong link? do you mean logger-sink.kamelet.yaml?

Reply:
The link I provided is wrong - I really meant jira-source kamelet, so I should change it to https://github.com/openshift-integration/kamelet-catalog/blob/master/jira-source.kamelet.yaml.